feature_tests/features/producer/producer-createDocumentPointer-success.feature

```diff
@@ -233,3 +261,39 @@ Feature: Producer Create Success scenarios
       | updated_on  | NULL                                    |
       | document    | <document>                              |
       | created_on  | 2023-05-02T12:00:00.000Z                |
+
+  @integration-only
+  Scenario: Validate a Document Pointer of type Mental health crisis plan with good URL
+    Given Producer "Aaron Court Mental Health NH" (Organisation ID "8FW23") is requesting to create Document Pointers
+    And Producer "Aaron Court Mental Health NH" is registered in the system for application "DataShare" (ID "z00z-y11y-x22x") with pointer types
+      | system                 | value     |
+      | http://snomed.info/sct | 736253002 |
+    And a Data Contract is registered in the system
+      | property             | value                  |
+      | name                 | Validate Content Url   |
+      | system               | http://snomed.info/sct |
+      | value                | 736253002              |
+      | version              | 1                      |
```

Comment:
Will we only ever be using a data contract for the snomed codes? If not then this table might get a little crowded as a method for populating the contract - not really an issue to worry about now though just thought id mention it if we check anything else

Reply:
hmmm dunno - I assume the design `C#system#value` was intended to allow for other code systems. I just wanted to test data contracts :p